tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -192,16 +205,88 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                                   string(name: 'RANCHER_K3S_VERSION', value: "${RANCHER_K3S_VERSION}"),
                                   string(name: 'BRANCH', value: branch) ]
 
-                      // Rancher server 1 is used to test RKE clusters
+                      goRKE2EC2Params = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN}"),
+                                  string(name: 'USER_TOKEN', value: "${USER_TOKEN}"),
+                                  string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"), 
+                                  string(name: 'TIMEOUT', value: "${TIMEOUT}"),
+                                  text(name: 'CONFIG', value: "${provision-config.yaml}"),
+                                  string(name: 'REPO', value: "https://github.com/rancher/rancher.git"),
+                                  string(name: 'BRANCH', value: branch),
+                                  string(name: 'TEST_PACKAGE', value: "provisioning/rke2"),
+                                  string(name: 'GOTEST_TESTCASE', value: "${rke2EC2TestCase}") ]
+
+                      goRKE2AZParams = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN}"),
```

Comment:
If these jobs were just provisioning, why would you need all of these env vars?

Reply:
I had these params so when I called the Go jobs, I just reference the specific params.

Reply:
I still don't quite understand. Is the goal to run the certification tests against the new provisioned clusters? Also if you are just running the go freeform job, these params are not necessary as the freeform job only needs the config.

Reply:
So the goal here is to essentially extend the coverage of the tests that the certification tests already do. We want to have RKE2/K3s run on EC2/Azure/custom clusters. So we decided using Go would be best as we have the freeform jobs to more so easily run them.

I can remove the params in that case and just have the config, I initially thought I needed all of the params in this ontag file.

Reply:
So we will have the same certification tests, but in Go? The part I'm missing is where the certification tests come into play. ie these cert jobs all run these tests against the cluster `[ "test_wl", "test_dns_record", "test_rbac","test_connectivity", "test_ingress",
                        "test_secrets", "test_registry", "test_service_discovery", "test_websocket"]`

So what you are setting is the params for the job you want to run, since the freeform job only takes the config file, the rest really won't do anything.

Reply:
Sorry, I'm not explaining it well. To clarify, we're not creating the same certification tests in Go. The end goal is to add new tests as mentioned in the original [task](https://github.com/rancher/qa-tasks/issues/429#issue-1283170650) that supplement the existing certification tests. The Go test additions would not be running certification tests, they are strictly just provisioning tests to add more coverage to the daily ontag runs.

Reply:
Got it thank you!! So yeah the other params are not necessary only the params that the go freeform job needs.

Reply:
So I don't know if you forgot to update, but `CATTLE_TEST_URL` and `ADMIN_TOKEN` are not needed for the go job

Reply:
I can remove those

Reply:
Resolved.